src/components/color-picker-hex-input/color-picker-hex-input.tsx

```diff
@@ -288,6 +242,42 @@ export class ColorPickerHexInput {
   //
   //--------------------------------------------------------------------------
 
+  private internalSetValue(
+    value: string | null,
+    oldValue: string | null,
+    // skipEqual = true,
```

Comment:
wats dis? skipEqual stuff. Can we remove it?

Reply:
Whoopsie. I'll remove. Copypasta from the picker's internal value setter. 🍝